lib/elixir/pages/getting-started/case-cond-and-if.md

```diff
@@ -113,24 +128,24 @@ iex> x
 1
 ```
 
-In said cases, if you want to change a value, you must return the value from the [`if`](`if/2`):
+You see the return value of the [`if`](`if/2`) expression as the resulting `2` here. To retain changes made within the [`if`](`if/2`) expression on the outer block you need to assign the returned value to a variable in the outer block.
 
 ```elixir
 iex> x = 1
 1
-iex> x = if true do
-...>   x + 1
-...> else
-...>   x
-...> end
+iex> x =
+...>   if true do
+...>     x + 1
+...>   else
+...>     x
+...>   end
 2
 ```
 
-> #### `if` is a macro {: .info}
->
-> An interesting note regarding [`if`](`if/2`) is that it is implemented as a macro in the language: it isn't a special language construct as it would be in many languages. You can check the documentation and its source for more information.
-
-If you find yourself nesting several [`if`](`if/2`) blocks, you may want to consider using [`cond`](`cond/1`) instead. Let's check it out.
+With all expressions returning a value there's also no need for alternative syntaxes
+like e.g. a ternary operator posing as an alternative to [`if`](`if/2`). You can
+use [`if`](`if/2`) directly – as we'll [show later](keywords-and-maps.md#do-blocks-and-keywords)
+there's even an inline notation for [`if`](`if/2`).
```

Comment:
In the guides we don't wrap lines.  Although I thought the paragraph a bit ambiguous. We say we don't need "alternative syntaxes" but we sort of link to an alternative syntax. So I chose to use the word "construct":

```suggestion
With all expressions returning a value there's also no need for alternative constructs, such as ternary operators posing as an alternative to [`if`](`if/2`). Elixir does include an inline notation for [`if`](`if/2`) and, as we will [learn later](keywords-and-maps.md#do-blocks-and-keywords), it is a syntactic variation on `if`'s arguments.
```